sopa/segmentation/stainings.py

```diff
@@ -105,8 +107,29 @@ def _run_patch(self, patch: Polygon) -> list[Polygon]:
             y=slice(bounds[1], bounds[3]),
         ).values
 
-        image = gaussian_filter(image, sigma=self.gaussian_sigma)
-        image = exposure.equalize_adapthist(image, clip_limit=self.clip_limit)
+        if len(self.channels) == 1:
```

Comment:
I think we don't need this "if-else" statement. The image always has a shape (C, Y, X), with sometimes C=1. For readability, I would keep only the "multi-channel" case below